modules/ROOT/pages/_partials/sgw-svr-compatibility.adoc

```diff
@@ -181,6 +181,15 @@ Compatibility Matrix
 
 |===
 
+[WARNING]
+====
+Starting from CBS 7.0, the `use_views` feature is no longer available.
+
+* SGW 3.1 will *still* run with `use_views` using a `default scope/default collection` configuration
+* SGW 3.1 will *not* run `use_views` using a `custom scope/custom collection` configuration
```

Comment:
'default scope/default collection' isn't code, so shouldn't be in backticks.

Not sure sure if 'custom' is the right word either?

Reply:
I'll check

Reply:
Before I do, though. I understand that once a `_dafault` scope and/or collection has been customised -  it has a new name - decided by the user. So not knowing what that is, I have used the term use the term custom.

I used the backticks for this section, as per the server docs.

Please let me know your thoughts or ideas. Thanks 

![image](https://github.com/couchbase/docs-sync-gateway/assets/111994975/f2eae769-367c-4a67-bad6-12718bc57e4d)

Reply:
So I will remove the backticks altogether as you suggest. And I'll change the custom to defined. How does that sound?

i.e 

- using a default scope/collection 
- tuning a defined scope/collection

??